src/lib.rs

```diff
@@ -18,10 +18,7 @@ mod lossy_pht;
 /// `Symbol`s are small (up to 8-byte) segments of strings, stored in a [`Compressor`][`crate::Compressor`] and
 /// identified by an 8-bit code.
 #[derive(Copy, Clone)]
-pub union Symbol {
-    bytes: [u8; 8],
-    num: u64,
-}
+pub struct Symbol(u64);
```

Comment:
going from union -> newtype struct shows up in the `cargo asm`. before symbol was getting spilled into stack, now it's passed in registers